docs/reference/db-config.rst

```diff
@@ -1,8 +1,35 @@
 :orphan:
 
-=======================
-db/config.cc properties
-=======================
+======================
+db/config.cc reference
+======================
 
-.. datatemplate:yaml:: ../_data/db_config.yaml
+Configuration properties
+------------------------
+
+.. dbconfigdatatemplate:: ../_data/db_config.yaml
+  :template: db_config.tmpl
+  :filter: Used
+
+
+Unused properties
+-----------------
+
+As ScyllaDB evolves, certain configuration properties may become obsolete or redundant.
+In our commitment to ensure smooth transitions, we have opted to keep this properties documented.
+Even though ScyllaDB does not use their values, we are maintaining them for anyone using older configurations that reference these options.
+
+Here is the list of unused properties:
+
+.. dbconfigdatatemplate:: ../_data/db_config.yaml
+  :template: db_config.tmpl
+  :filter: Unused
+
+Invalid properties
+------------------
+
+Todo: Description
```

Comment:
What's the difference between invalid and unused values?

Reply:
if an option is marked "Invalid", if it is specified in the config yaml file, a log message is printed with warning level when the scylla service starts. while the "Unused" options are just silently ignored.